sample_settings.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
   debug: false,
   usePlaceholderIssuesForMissingIssues: true,
   useReplacementIssuesForCreationFails: true,
+  useIssuesForAllMergeRequests: false,
```

Comment:
Could you add some documentation in the `readme.md` what this setting does?

Reply:
Thanks - good idea. Done.